src/notebooks/controllers/vscodeNotebookController.ts

```diff
@@ -83,11 +81,12 @@ import {
 } from '../../kernels/types';
 import { InteractiveWindowView } from '../constants';
 import { CellExecutionCreator } from '../execution/cellExecutionCreator';
-import { isJupyterNotebook, traceCellMessage, updateNotebookDocumentMetadata } from '../helpers.node';
-import { KernelDeadError } from '../../platform/errors/kernelDeadError.node';
-import { DisplayOptions } from '../../kernels/displayOptions.node';
-import { sendNotebookOrKernelLanguageTelemetry } from '../../platform/common/utils.node';
+import { isJupyterNotebook, traceCellMessage, updateNotebookDocumentMetadata } from '../helpers';
+import { KernelDeadError } from '../../platform/errors/kernelDeadError';
+import { DisplayOptions } from '../../kernels/displayOptions';
+import { sendNotebookOrKernelLanguageTelemetry } from '../../platform/common/utils';
 import { ConsoleForegroundColors, TraceOptions } from '../../platform/logging/types';
+import { KernelConnector } from '../../kernels/kernelConnector';
 
 export class VSCodeNotebookController implements Disposable {
```

Comment:
This file was the reason for this PR. And just getting this to build caused all of these other changes :)